
<launch>

	<node name="move" pkg="udm_project_control" type="indirect_kin_service.py">
		<param name="group_name" value="fl"/>
	</node>

</launch>
